modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx

```diff
@@ -332,7 +318,7 @@ function Form (props) {
               buttonClass={classes.titleButton}
             />
           </Typography>
-          <Breadcrumbs separator="·">
+          <Breadcrumbs separator=".">
```

Comment:
This needs to be reverted
```suggestion
          <Breadcrumbs separator="·">
```